### PR TITLE
Export Kind logs to ease debugging

### PR DESCRIPTION
**Pull request by collivier**
It also deletes cluster after Oran testing

## Description

No logs after failures in gates

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [X] Verified all A/C passes
     * [ ] develop
     * [X] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [X] Kind cluster

## Types of changes:
 - [X] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

### CODE REVIEW

**Review by martin-mat — APPROVED** [review]
lgtm

**Review by Smitholi67 — APPROVED** [review]
lgtm

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -208,10 +208,17 @@ jobs:
       if: ${{ always() }}
       run: |
         source cluster.env
+        kind export logs --name $CLUSTER /tmp/outpur-dir
```
Comment (kosstennbl) [inline .github/workflows/actions.yml:211]: Typo: outpur-dir

Comment (collivier) [reply .github/workflows/actions.yml:211]: oh well catch.

**Review by kosstennbl — APPROVED** [review]
LGTM

**Pull request merged**